python/pyarrow/_rados.pyx

```diff
@@ -22,6 +22,17 @@ from pyarrow.lib cimport *
 from pyarrow.lib import frombytes, tobytes
 
 cdef class RadosParquetFileFormat(FileFormat):
+    """
+    A ParquetFileFormat implementation that offloads the fragment
+    scan operations to the Ceph OSDs
+
+    Parameters
+    ---------
+    ceph_config_path: The path to the ceph config file.
+    data_pool: Name of the CephFS data pool.  
+    user_name: Username to access the Ceph data cluster.
+    cluster_name: Name of the data cluster.
```

Comment:
We can just say the cluster instead of data cluster. There is no concept of data cluster/metadata cluster. There is a single Ceph cluster with 2 types of pools. data pools and metadata pools within the same set of nodes (same cluster).

Reply:
Done.